src/main/resources/application.properties

```diff
@@ -1 +1 @@
-spring.datasource.url=jdbc:h2:mem:test
+
```

Comment:
let’s configure access to h2 database through the console in `application.properties`
you should be able to view your h2 DB from browser